src/lib/accessibility/Behaviors/Tree/treeTitleBehavior.ts

```diff
@@ -0,0 +1,15 @@
+import { Accessibility } from '../../types'
+
+/**
+ * @specification
+ * Adds attribute 'aria-expanded=true' based on whether the folowing subtree is expanded or not.
+ */
+const treeTitleBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      ...(props.hasSubtree && { 'aria-expanded': props.open ? 'true' : 'false' }),
```

Comment:
for some reason this does not seem to work correcttly. In the screenshot below, first opened node is highlighted, but it has `aria-expanded='false'`:
![image](https://user-images.githubusercontent.com/13742664/48888709-da6b2d80-ee33-11e8-9fb6-5643c4f874a4.png)

Reply:
Fixed